WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -467,15 +527,29 @@ public void onSuccess(long personID, int localTableBlogId) {
 
             @Override
             public void onError() {
+                int errorMessageRes;
+                switch (personType) {
+                    case USER:
+                        errorMessageRes = R.string.error_remove_user;
+                        break;
+                    case VIEWER:
+                        errorMessageRes = R.string.error_remove_viewer;
+                        break;
+                    default:
+                        errorMessageRes = R.string.error_remove_follower;
```

Comment:
This message doesn't seem generic as it is follower related so, I'd recommend replacing the `default:` with the couple of `case`s that correspond to this message. Thanks!